examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+
+## Prerequisites
+This tutorial presumes that the repository in https://github.com/hyperledger-labs/fabric-private-chaincode has been installed as per [README.md](https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/README.md#requirements) in `FPC-INSTALL-DIR` which is in the `$GOPATH` folder. Additionally the [HelloWorld] example (https://github.com/hyperledger-labs/fabric-private-chaincode/blob/master/examples/README.md) has been completed.
+
+## Develop client gateway go script
+* Create a folder named `go_app`  in `FPC-INSTALL-DIR/examples/helloworld`.
+```bash
+cd $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode/examples/helloworld
+mkdir go_app
+cd go_app
+touch helloworld.go
+```
+
+* Define the go package and import the necessary libraries that will be used. `os` to access operating system variables and two libraries that enable us to access the fabric gateway and call the FPC chaincode from the gateway.
+
+```go
+package main
+
+import (
+	"os"
+
+	fpc "github.com/hyperledger-labs/fabric-private-chaincode/client_sdk/go/pkg/gateway"
+	"github.com/hyperledger-labs/fabric-private-chaincode/integration/client_sdk/go/utils"
+	"github.com/hyperledger/fabric-sdk-go/pkg/gateway"
+	"github.com/hyperledger/fabric/common/flogging"
+)
+```
+
+* We will also define several variables. `network` represents the Fabric network and `contract` our fabric private chaincode developed previously. We also are using a variable to be able to capture the errors that the different functions might return.
+
+```go
+var (
+	network  *gateway.Network
+	contract fpc.Contract
+	err      error
+	logger   = flogging.MustGetLogger("helloworld")
+)
+```
+* We will then add the main function which will execute our code. The environent variable `CC_ID` represents the FPC chaincode already installed on the channel and will be set from the bash script we will create to run evenrything. The Fabric network we are setting up has only one channel called `mychannel` which we will connect to, using the `SetupNetwork` go method provided in the `utils` library we imported before. Then we will access the FPC chaincode and call the `SubmitTransaction` go method that uses the first parameter as the chaincode method to invoke, the second as the name of the asset and the third as the value. In this case we will be storing 100 as the value of `asset1`  invoking the `storeAsset` method.
+
+```go
+func main() {
+	ccID := os.Getenv("CC_ID")
+	logger.Infof("Environment variable CC_ID: %v", ccID)
+	// get network
+	network, err = utils.SetupNetwork("mychannel")
```

Comment:
let's do somthing with the error ... I think a panic would be simple and ok here.

Reply:
Also, should we maybe also read the channel name from ENV?

Reply:
is it stored already in an ENV? or would we have to configure like with CC_ID?

Reply:
It think it is not - so exporting as we do for CC_ID would work I guess